### PR TITLE
[SE] Make coverage build runnable

### PR DESCRIPTION
**Pull request by zhoonit**
This patch is a workaround to make coverage build runnable for now.
There will be following patches to enable the commented lines.

See also #997

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

### CODE REVIEW

File: test/unittest/meson.build [hunk 1]
```diff
@@ -34,13 +34,10 @@ test_target = [
   'unittest_nntrainer_modelfile',
   'unittest_nntrainer_models',
   'unittest_nntrainer_graph',
-  'unittest_nntrainer_appcontext'
+  'unittest_nntrainer_appcontext',
+  'unittest_nntrainer_profiler'
 ]
 
-if get_option('enable-profile')
```
Comment (kparichay) [inline test/unittest/meson.build:40]: any reason to remove this?

Comment (zhoonit) [reply test/unittest/meson.build:40]: eventhough enable-profile is off, unittest is runnable, without this the test coverage is 0% for the profiler

Comment (kparichay) [reply test/unittest/meson.build:40]: then how about instead of removing this, enable profile in the CI build (with unit_test).

Comment (zhoonit) [reply test/unittest/meson.build]: Updated

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Review by lhs8928 — APPROVED** [review]
LGTM

**Review by kparichay — APPROVED** [review]
LGTM

**Pull request merged**